doc/source/Reference/ScriptingReference/Metadata/index.md

```diff
@@ -0,0 +1,115 @@
+Metadata
+========
+
+Gaffer's UIs for plugs and nodes are defined using a metadata convention. This
+makes it easy to customise the UI for a specific node type, or even for a
+specific node instance. This document provides a reference for the most common
+metadata items.
+
+General
+-------
+
+```eval_rst
+=================== =============================== ============================
+Name                Purpose                         Example Values
+=================== =============================== ============================
+label               Label used instead of plug name "My Label"
+description			Describes the purpose of a      "Turns on the thingammajig"
+                    node or plug.
+userDefault         Overrides the default value     10.5
+preset:<name>       Specifies a named preset value  "preset:Max", 1
```

Comment:
I've added icon, but I've left presetNames/presetValues out, because they're really a workaround for a lack of more dynamic metadata, so I'd rather not advertise them for now.